docroot/modules/custom/cs_generated_content/generated_content/media/civictheme_image.inc

```diff
@@ -34,9 +34,14 @@ function cs_generated_content_generated_content_create_media_civictheme_image()
   $total_files_count = 50;
 
   $filename = sprintf('Demo image file %s', $helper::staticSentence(2));
-  $file = $helper::createFile(GeneratedContentAssetGenerator::ASSET_TYPE_PNG, [
-    'filename' => $filename,
-  ], GeneratedContentAssetGenerator::GENERATE_TYPE_STATIC);
+  $files = [];
+  for ($i = 1; $i < 4; $i++) {
+    $files[] = $helper::createFile(GeneratedContentAssetGenerator::ASSET_TYPE_PNG, [
```

Comment:
@AlexSkrypnyk Could you please confirm if we can use this?

Reply:
we cannot use random in any of these cases as this will break visual regression testing